README.md

```diff
@@ -27,6 +27,16 @@ $ npx require-so-slow got@rc
 $ npm require-so-slow -o lodash.trace.json lodash
 ```
 
+You can also preload require-so-slow from node:
+```shell
+npm i -D require-so-slow
+
+# Traces the entire execution and outputs to require-so-slow.trace:
+node -r require-so-slow [moduleOrEntrypoint]
+
+# You can specify the output file with an environment variable:
+TRACE_OUTFILE=./rss.trace node -r require-so-slow [moduleOrEntrypoint]
```

Comment:
My suggestion would be for the default example to be the easier one to get stared with / remember. Maybe have two examples, or mention the env. var. in the text?